src/SDKs/DevSpaces/Management.DevSpaces/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Reflection;
+using System.Resources;
+
+[assembly: AssemblyTitle("Microsoft Azure Dev Spaces Management Library")]
+[assembly: AssemblyDescription("Provides management operations support for Azure Dev Spaces.")]
+
+[assembly: AssemblyVersion("1.0.0.0")]
+[assembly: AssemblyFileVersion("1.0.0.0")]
```

Comment:
please start your first release versioning scheme at 0.9.0.0 for your first preview release